packages/runtime/src/widgets/Form/__tests__/MultiSelect.test.tsx

```diff
@@ -196,19 +205,45 @@ describe("MultiSelect Widget", () => {
     );
 
     userEvent.click(screen.getByRole("combobox"));
-    fireEvent.click(screen.getByTitle("Option 2"));
-    fireEvent.click(screen.getByTitle("Option 4"));
+    userEvent.click(screen.getByTitle("Option 2"));
+    userEvent.click(screen.getByTitle("Option 4"));
     userEvent.click(screen.getByRole("combobox"));
 
+    // Wait for the combobox to reflect the selected values
+    await waitFor(() => {
```

Comment:
Why was this test passing before, and how does this change prevent this issue from occurring?

Reply:
Yes, it found multiple elements with same text, one in input area where selected values will show and one from dropdown list, I confirmed this with the element selector and both are different element with same text. I also tried to close the dropdown list after selecting the option with fireEvent and userEvent. the dropdown list is closing but it is still part of document